### PR TITLE
Adding email stats documentation

### PR DESCRIPTION
**Pull request by EvanEPrice**
Don't merge this yet, there might be upcoming changes.

### CODE REVIEW

File: docs/kb/api-version-3/emails.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Reading Emails
+# Reading Emails
```
Comment (et) [inline docs/kb/api-version-3/emails.md]: typo?

**Comment by et** [comment]
👍

**Comment by jusroberts** [comment]
Make sure you do this change to the V4 version as well. Probably just copy-pasting. https://github.com/Pardot/api-docs/blob/master/docs/kb/api-version-4/emails.md

**Comment by humeansacrifice** [comment]
@EvanEPrice where are we at with this?

**Pull request merged**